charset_normalizer/cd.py

```diff
@@ -5,6 +5,7 @@
 from typing import Dict, List, Optional, Tuple
 
 from .assets import FREQUENCIES
+from .constant import KO_NAMES, ZH_NAMES
```

Comment:
I am curious about the names. What does `KO` and `ZH` stand for?

Reply:
These are two-letter ISO language codes: Korean and Chinese.

Reply:
Thanks! Indeed.